projects/qtaf/docs/sections/plugins/xray/Configuration.md

```diff
@@ -0,0 +1,1383 @@
+
+***Type***
+: `string`
+
+***Default***
+: :fontawesome-solid-server:{ title="Xray server" .xray-icon } `#!json "FAIL"`
+: :fontawesome-solid-cloud:{ title="Xray cloud" .xray-icon } `#!json "FAILED"`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "customStatus": {
+              "test": {
+                "failed": "ERROR"
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_FAILED="ERROR"
+        ```
+
+<hr/>
+
+##### `passed`
+
+The Xray status of passed tests.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_PASSED`
+
+***Type***
+: `string`
+
+***Default***
+: :fontawesome-solid-server:{ title="Xray server" .xray-icon } `#!json "PASS"`
+: :fontawesome-solid-cloud:{ title="Xray cloud" .xray-icon } `#!json "PASSED"`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "customStatus": {
+              "test": {
+                "passed": "DONE"
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_PASSED="DONE"
+        ```
+
+<hr/>
+
+##### `pending`
+
+The Xray status of pending tests.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_PENDING`
+
+***Type***
+: `string`
+
+***Default***
+: `#!json "EXECUTING"`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "customStatus": {
+              "test": {
+                "pending": "EXECUTING"
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_PENDING="EXECUTING"
+        ```
+
+<hr/>
+
+##### `skipped`
+
+The Xray status of skipped tests.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_SKIPPED`
+
+***Type***
+: `string`
+
+***Default***
+: `#!json "TODO"`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "customStatus": {
+              "test": {
+                "skipped": "TODO"
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_SKIPPED="TODO"
+        ```
+
+<hr/>
+
+#### `testExecutionIssue`
+
+The Jira status of test execution issues created during upload.
+
+<hr/>
+
+##### `failed`
+
+The Jira status to transition test execution issues to if tests failed.
+If the status is `#!json null`, the issue will have the default issue status of issues created in the project.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_EXECUTION_ISSUE_FAILED`
```

Comment:
It must be TESTEXECUTIONISSUE instead of TEST_EXECUTION_ISSUE
This error occurs multiple times in the document

Reply:
Fixed in c881120011c3b41aa0c792e92917b7691403ddf5.